src/zeroconf/_core.py

```diff
@@ -240,7 +240,7 @@ async def async_wait_for_start(self) -> None:
             raise NotRunningException
 
     @property
-    def listeners(self) -> List[RecordUpdateListener]:
+    def listeners(self) -> Set[RecordUpdateListener]:
         return self.record_manager.listeners
```

Comment:
I'm not sure anything actually uses this but there is always someone.

We can always wrap it in `list()` in a followup release if this is causing breaking for someone if they open an issue, but that seems very unlikely